### PR TITLE
feat!: support gas price and limit for cashout

### LINKED ISSUE #310: Cashout cheque gas-price/limit
The cashout endpoint supports specifying gas price and gas limit, we currently do not expose these options in `BeeDebug`. FYI. these should be `BigInt`s

### PR DESCRIPTION
**Pull request by AuHau**
Closes #310 

Honestly, I would like to refactor the `cashoutLastCheque` to return the `transactionHash` directly and not wrapped in an object as is now, but it would be breaking change. What do you think?

### CODE REVIEW

File: src/bee-debug.ts [hunk 1]
```diff
@@ -25,7 +25,9 @@ import type {
   NodeAddresses,
 } from './types'
 import { assertBeeUrl, stripLastSlash } from './utils/url'
-import { assertAddress, assertInteger } from './utils/type'
+import { assertAddress, assertInteger, assertNonNegativeInteger } from './utils/type'
+import { CashoutOptions } from './types'
+import { add } from 'husky'
```
Comment (nugaon) [inline src/bee-debug.ts:30]: not used import
```suggestion
```

**Review by nugaon — APPROVED** [review]
looks good and I second your suggestion about the unwrapped return value.

**Comment by nugaon** [comment]
Also, there is unused import in `bee-class.spec.ts`  (`BeeError`), and it has type issue for the given `gasPrice` in line 156

**Comment by AuHau** [comment]
Thanks for the feedback, that is my trigger happy IDE 😅

**Pull request merged**